canvas_sdk/effects/banner_alert/remove_banner_alert.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any
+
+from canvas_sdk.effects.base import EffectType, _BaseEffect
+
+
+class RemoveBannerAlert(_BaseEffect):
+    """
+    An Effect that will remove/inactivate a banner alert in Canvas.
+    """
+
+    class Meta:
+        effect_type = EffectType.REMOVE_BANNER_ALERT
```

Comment:
I changed these to use the constant rather than the string "REMOVE_BANNER_ALERT"